CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Android Card Form Release Notes
 
+## unreleased
+
+* Deprecate `useDialogForExpirationDateEntry()`
+* Changed default expiration date keyboard to use a system numeric keyboard
+* Updated cvv keyboard to be numeric only
```

Comment:
This is a nitpick, but can we make the changelog entries (this one and the line above) present tense?

Reply:
Yeah def def